CRM/Speakcivi/Page/Lang.php

```diff
@@ -4,7 +4,7 @@
 
 class CRM_Speakcivi_Page_Lang extends CRM_Speakcivi_Page_Post {
 
-  private $urlLanguageForm = '/form/preferred-language';
+  private $urlLanguageForm = '/language-switch';
```

Comment:
Looks fine. However should't be also "change" verb? finally to '/language-change' as above in subject "Language change from..."

Reply:
Fine - reworked the URLs for all translations to /language-change.

Reply:
in 93ce276a57b26163a8a9a30fc2b6f142c19365e3